scripts/jenkins/Jenkinsfile

```diff
@@ -47,6 +47,7 @@ CMAKE_FLAGS_BASE = [
     'SITE': '${STAGE_NAME}',
     'KDB_DB_SYSTEM': '${HOME}/.config/kdb/system',
     'KDB_DB_SPEC': '${HOME}/.config/kdb/spec',
+    'XDG_CONFIG_DIRS': '${HOME}/.config/xdg/system',
```

Comment:
The idea of this build job is to replicate https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html

which means to directly write to "${HOME}/.config", "/etc/xdg" and so on. The XDG env vars, like XDG_CONFIG_DIRS, are set within the resolver tests in tests/shell/check_resolver.sh, so they need not to be set outside.

Reply:
should we just make /etc/xdg globally writable? same for /etc/kdb instead of setting it to the home directory of the user?

Reply:
Sorry, seems like the old test has set the environment variables "XDG_CONFIG_HOME and
 XDG_CONFIG_DIRS". It is fine if we replicate that. Only for the spec we need some workaround (like changing CMAKE_INSTALL_PREFIX) because of #734.

Reply:
I am afraid I do not follow:
The old job did not set any of those variables. It just calls https://github.com/ElektraInitiative/libelektra/blob/master/scripts/configure-xdg.

Reply:
https://build.libelektra.org/jenkins/job/elektra-gcc-configure-xdg/configure has:

```
export XDG_CONFIG_HOME="$WORKSPACE/user"
export XDG_CONFIG_DIRS="$WORKSPACE/system"
mkdir build
cd build
../scripts/configure-xdg -DCMAKE_INSTALL_PREFIX=$WORKSPACE/local -DBUILD_DOCUMENTATION=OFF ..
run-make

run-make run_all
run-make run_memcheck
```

And yes, it would be good if we can continue to use the script.